### ISSUE TITLE
`woocommerce-tokenization-form` script should be loaded `in_footer`

### ISSUE THREAD

**Issue by cawa-93** [woocommerce/woocommerce#40489]
https://github.com/woocommerce/woocommerce/blob/d8aa52d0a7e1907346223b9e421d411d479b0770/plugins/woocommerce/includes/abstracts/abstract-wc-payment-gateway.php#L457-L462

At the moment, this script inserts into the header (and forces the insertion of j into the header), which blocks the rendering of the page. However, the script itself runs its logic only after the home page is fully loaded. Therefore, I think it is advisable to move the loading of this script to the footer.

**Comment by vbelolapotkov** [comment]
@pierorocca sending this to you for prioritization. If I understand correctly, the tokenization form manages saved payment methods on checkout. If there is no other good reason to load it early, then the suggested improvement can slightly improve initial loading UX.

**Comment by pierorocca** [comment]
@vbelolapotkov Wordpress 8.0 has enabled asynch loading so it would be worth auditing checkout for opportunities to improve performance and have as much as possible as non-blocking on load. Tagging @frosso team Heisenberg for triaging.

**Comment by flakas** [comment]
Hi @cawa-93 - thanks for the report.

I know this has been open for a long time, and I appreciate your patience. We're currently going through some older WooCommerce issues, identifying what still needs fixing, and tidying up what's already fixed or out of scope.

I've taken a look and marked this ready for the engineering team to pick up.

What I'm seeing: The tokenization form script is still enqueued with the final wp_enqueue_script argument set to false, which means it is not loaded in the footer. The reported optimization opportunity is still present on trunk.
